packages/os/os.spec

```diff
@@ -89,6 +92,7 @@ Requires: %{_cross_os}storewolf
 Requires: %{_cross_os}sundog
 Requires: %{_cross_os}xfscli
 Requires: %{_cross_os}thar-be-settings
+Requires: %{_cross_os}configure-snapshotter
```

Comment:
Are you adding this here because the snapshotter is independent of the SOCI package or containerd? If that's the case, this will be better at `release.spec` instead of `os.spec`.

`os.spec` is mostly used to compile first party Rust. `release.spec` is where we put things that should exist in all variants.

Reply:
+1, definitely put this in `release.spec`. I cringe whenever anything triggers a rebuild for `os`.